core/src/components/input/test/basic/input.e2e.ts

```diff
@@ -63,6 +62,45 @@ configs().forEach(({ title, screenshot, config }) => {
         await expect(input).toHaveScreenshot(screenshot(`input-with-clear-button-stacked`));
       });
     });
+    test.describe('input click behaviors', () => {
+      test('should not have visual regressions', async ({ page }) => {
+        await page.setContent(
+          `
+          <ion-input
+            label="Label"
+            clear-input="true"
+            value="Text"
+          ></ion-input>
+        `,
+          config
+        );
+        const input = page.locator('ion-input');
+        input.dblclick();
+        // Validates the display of an input with a clear button.
+        await expect(input).toHaveScreenshot(screenshot(`input-dbclick`));
+      });
+      test('should not have visual regressions with stacked label', async ({ page, browserName }) => {
+        if (browserName === 'firefox') {
+          //TODO: Figure out why it's not working for firefox
+          test.skip();
+        }
```

Comment:
Once I used the native input to click on then the Firefox tests were passing as expected.

```suggestion
```